webpack/assets/javascripts/compute_resource/ovirt.js

```diff
@@ -68,6 +68,17 @@ export function instanceTypeSelected(item) {
         $('[id$=_ha]').prop('checked', result.ha);
         const instanceTypeSelector = $('#host_compute_attributes_instance_type');
 
+        if (result.name != null) {
+          $('[id$=_memory]').attr('disabled', 'disabled');
+          $('[id$=_cores]').attr('disabled', 'disabled');
+          $('[id$=_sockets]').attr('disabled', 'disabled');
+          $('[id$=_ha]').attr('disabled', 'disabled');
```

Comment:
If you would like, you can do something like `[ $('[id$=_memory]'), $('[id$=_cores]'), $('[id$=_sockets]'), $('[id$=_ha]') ].forEach( ()=>
   $(this).attr('disabled', 'disabled')
})`